### DISCUSSION TITLE
Answer from GitHub Support about the feed

### DISCUSSION THREAD

**Discussion by falsandtru** [community/community#37616] [Other Feature Feedback, Questions, & Ideas]
I report the answer from GitHub Support. I wish everyone who sees this will be satisfied.

> GitHub Support commented
Hi falsandtru,
>
>Thanks for contacting GitHub Support! 😊
>
>I can certainly clear the following feed for you. This will clear all public and private events from your "Following" feed completely. Would you like me to continue?
>
>I am looking forward to hearing back from you 😊
>
>Best wishes,
Pedro
>
>🌯 Find answers to common questions and learn with other GitHub users in our new [GitHub Community Forum](https://github.community/) 🌯

Related: #37572

**Comment by falsandtru** [comment]
I don't know why he is so happy when we are in so much trouble here. He doesn't seem to see here.

**Comment by hmmhmmhm** [comment]
This a/b test is the worst other test I've ever had.

I can only see one thing that other people pressed star. (I have 300 following developers and originally identified a list of open sources that more than 100 others are interested in each week.)

![image](https://user-images.githubusercontent.com/38493722/198556710-958fd528-c5f8-4fdf-b867-c446bff1701b.png)

The repos recommended by Github's algorithms are generally only too low on interest.
I want to figure out what developers I'm interested in are interested in, rather than artificial recommendations.

At least these two pieces of information should be separate.

**Reply by falsandtru** [reply]
Anyway no way to restore the previous activity. We can only choose Feed or Nothing.

**Reply by hmmhmmhm** [reply]
Why should I be chosen to suffer this? Can't I transmit this pain to my friend? 
I feel like I've become a zombie who doesn't have the ability to infect.

**Reply by sinclairzx81** [reply]
@hmmhmmhm I hear you. I'm currently on the AB test and have basically lost all notification visibility for the projects I maintain and contribute to. If I enable any one of those `Filters`, it just results in recommendation spam. I've currently got everything disabled resulting in a `That's all for now` message. Not very useful.

Honestly, I don't mind Github adding some form of recommendation system, but I can't understand their decision to replace critical functionality with some random recommendation feed. Surely the AB test could have included the previous feed in the `Filters` as not to disrupt those forced into AB testing.

Are GH really not responding to concerns here?

**Reply by sinclairzx81** [reply]
@falsandtru @hmmhmmhm https://github.com/orgs/community/discussions/37693

**Reply by falsandtru** [reply]
Looks like just a source (fragment) of an old dashboard. Not made for direct viewing.

**Comment by falsandtru** [comment]
Has anyone else inquired through the contact form?
